docusaurus/docs/dev-docs/quick-start.md

```diff
@@ -166,71 +183,55 @@ You have just created a basic data structure for your Strapi project! You can ke
 
 ## ☁️ Part C: Deploy to Strapi Cloud
 
-Now that your beautiful first Strapi project is working locally, it's time for the world to see it live! The most straightforward way to host your project is to use Strapi Cloud. Before deploying to Strapi Cloud, you will need to host your Strapi project on an online repository — we will use GitHub.
+Now that your beautiful first Strapi project is working locally, it's time for the world to see it live! The most straightforward way to host your project is to use Strapi Cloud: Deploying your project on Strapi Cloud is done with a single command! 🚀
 
-<details style={detailsStyle}>
-<summary style={summaryStyle}>Step 1: Host the code of your Strapi project on GitHub</summary>
+To deploy your project on Strapi Cloud, in your terminal:
 
-### Step 1: Host the code of your Strapi project on GitHub
+1. If the server for your local Strapi project is running, which should be the case if you followed this tutorial so far, press `Ctrl-C` to stop the server.
+2. Ensure you are in the folder of your Strapi project (if needed, run for instance `cd my-strapi-project` to reach this folder), and run the following command:
 
-Create a new GitHub repository and push the code of your Strapi project to this repository. If you're not already familiar with GitHub, the togglable content below should get you started 👇
+    <Tabs groupId="yarn-npm">
 
-<details>
-<summary>Steps required to push your Strapi project code to GitHub:</summary>
+    <TabItem value="yarn" label="Yarn">
 
-1. In the terminal, ensure you are still in the `my-project` folder that hosts the Strapi project we created. If you followed this tutorial closely so far, we should still be there.
-2. Run the `git init` command to initialize git for this folder.
-3. Run the `git add .` command to add all modified files to the git index.
-4. Run the `git commit -m "Initial commit"` command to create a commit with all the added changes.
-5. Log in into your GitHub account and [create a new repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/quickstart-for-repositories). Give the new repository a name, for instance `my-first-strapi-project`, and remember this name.
-6. Go back to the terminal and push your local repository to GitHub:
+      ```sh
+      yarn strapi deploy
```

Comment:
Could you please double-check the commands for me? My draft was done using the QA/experimental builds and commands differ. 🙏

Reply:
LGTM